### ISSUE TITLE
Hubble service should not be started as soon as you installs it

### ISSUE THREAD

**Issue by anuragpaliwal80** [hubblestack/hubble#294]

**Comment by basepi** [comment]
What are your thoughts on restarting the service on upgrade?

**Comment by anuragpaliwal80** [comment]
I think if possible lets keep in stop state and let user starts it explicitly when they want.

**Comment by fossam** [comment]
@anuragpaliwal80 Have you considered the risk of users forgetting to do this? 
And as @basepi suggested, what about a restart?

Basically, let's have a more spelled out reasoning for why it should be this or that way.

**Comment by anuragpaliwal80** [comment]
Basically I was thinking of having same behavior for users. Either they install fresh build or upgrades it, they should explicitly start the service without assuming that it will be started automatically in case of upgrade.
But I agree with @basepi as both cases have it's pros and cons.

**Comment by basepi** [comment]
Yeah, for us we have our default_hubble index which is pre-populated in our internal packages. So having it start on install is nice so we can have visibility into the host even if it the user forgets to configure it. So I'm torn...
